src/NuGet.Services.EndToEnd/Support/EnvironmentSettings.cs

```diff
@@ -26,6 +26,11 @@ public static class EnvironmentSettings
         /// </summary>
         public static string ConfigurationName => GetEnvironmentVariable("ConfigurationName", required: true);
 
+        /// <summary>
+        /// The path to the signed package that should be used for tests.
+        /// </summary>
+        public static string SignedPackagePath = GetEnvironmentVariable("SignedPackagePath", required: false);
```

Comment:
is this necessary because of the way we create the package? Since we are moving away from Environment Variables for config, it would be better if this could be configured in a json file, like everything else.

Reply:
Yup, the VSTS build itself sets the environment variable if it has the `GenerateSignedPackage` step. If you don't have that step, the environment variable won't exist and the signed package tests won't run.